modules/luci-base/htdocs/luci-static/resources/network.js

```diff
@@ -3880,7 +3902,17 @@ WifiNetwork = baseclass.extend(/** @lends LuCI.network.WifiNetwork.prototype */
 	 * with this network.
 	 */
 	getAssocList: function() {
-		return callIwinfoAssoclist(this.getIfname());
+		var tasks = [];
+		var ifnames = Array.prototype.concat.apply(
```

Comment:
I'd prefer this simpler expression: `var ifnames = [ this.getIfname() ].concat(this.getVlanIfnames());`